src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -110,8 +103,21 @@ class CosmasGoogleCloudService(
             res.addAll(oldWindow.take(min(windowMaxSize - 1, oldWindow.size)))
             return res
         }
+
+        fun cemeteryName(info: ProjectInfo) = "${info.projectId}-cemetery"
```

Comment:
I think these functions can be moved to ServiceFilesMediator file, because they are only used there (besides tests)

Reply:
ok